topics:
  inputs:
    octomap: 
      name: /map/octomap
      type: octomap_msgs/Octomap
    chassis_pose:
      name: /chassis/pose
      type: geometry_msgs/PoseStamped
    target_pose:
      name: /task_mgr/target_object/pose
      type: geometry_msgs/PoseStamped
    joint_states:
      name: /arm/joint_states
      type: sensor_msgs/JointState
    point_cloud:
      name: /sensors/point_cloud
      type: sensor_msgs/PointCloud2
  outputs:
    arm_trajectory:
      name: /arm/trajectory
      type: arm_planner/ArmTrajectory
    end_pose:
      name: /arm/end_pose
      type: geometry_msgs/PoseStamped



